src/libs/Anthropic/openapi.yaml

```
@@ -929,6 +1033,30 @@ components:
       required:
         - type
+    ErrorEvent:
+      type: object
+      description: An error event in a streaming conversation.
+      properties:
+        type:
+          $ref: "#/components/schemas/MessageStreamEventType"
+        error:
+          $ref: "#/components/schemas/Error"
+      required:
+          - type
```

yamllint

[warning] wrong indentation: expected 8 but found 10

(indentation)